src/dmd/dsymbolsem.d

```diff
@@ -5426,6 +5398,50 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     }
 }
 
+/*******************************************
+ * Add members of EnumDeclaration to the symbol table(s).
+ * Params:
+ *      ed = EnumDeclaration
+ *      sc = context of `ed`
+ *      sds = symbol table that `ed` resides in
+ */
+void addEnumMembers(EnumDeclaration ed, Scope* sc, ScopeDsymbol sds)
```

Comment:
in the effort to have AST not depend on semantic, please declare this in `denum.d` rather than `dsymbolsem.d`

Reply:
I thought the point was to have denum not depend on semantic.

Reply:
I confess to being conflicted as to where it goes.